src/js/components/StarRatingComponent.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+export default class StarRatingComponent extends React.Component{
```

Comment:
to jest praktycznie ten sam komponent co Start. Jeżeli jest taka potrzeba zrób jeden generyczny komponent. Nie możemy duplikować przecież kodu :)